charts/seed-controlplane/charts/kube-apiserver/values-test.yaml

```diff
@@ -0,0 +1,8 @@
+watchCacheSizes:
```

Comment:
Nope. I thought, it would be good to test, that the chart can be correctly rendered, if the values are added (default values are commented out).
But as you said, we should generally think about a proper way to handle default values in our charts. So I don't mind removing it in this PR :)

Reply:
Hm, yeah, right. I would probably do that separately, i.e., we should comment all Helm chart default values and set them explicitly in code. Separate topic, though 😄

Reply:
Done, removed the file.